### PR TITLE
FailoverClusters Bulk Fixup (3/4) - Update example formatting

### PR DESCRIPTION
**Pull request by robinharwood**
Bulk updating example formatting before content updates

### CODE REVIEW

**Comment by robinharwood** [comment]
Hi @michaeltlombardi,

I believe this PR is release ready (minus on comment for you above) if you would be kind enough to review and if appropriate sign-off/merge 😊

Please let me know if you have any questions or if I need to apply any other updates.

Thanks Robin

File: docset/winserver2022-ps/failoverclusters/Remove-ClusterCheckpoint.md [hunk 1]
```diff
@@ -36,20 +36,13 @@ local server.
...
-Are you sure you want to remove registry checkpoint 'software\clusname' on resource 'Cluster Name'? 
-
-
-[Y] Yes  [N] No  [S] Suspend  [?] Help (default is "Y"):Y
-```
-
-This example removes the registry checkpoint called software\clusname for the resource named Cluster
-Name.
+This example removes the registry checkpoint called `software\clusname` for the resource named
+`Cluster Name` without user confirmation.
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/failoverclusters/Remove-ClusterCheckpoint.md:45]: Without the output, it's not clear to me where `software\clusname` is coming from, which might be confusing for a reader.

File: docset/winserver2022-ps/failoverclusters/Remove-ClusterGroup.md [hunk 2]
```diff
@@ -32,30 +32,30 @@ resources first, or specify the **RemoveResources** parameter.
...
 
-```
-PS C:\> Remove-ClusterGroup -Name MyFileServer -Force -RemoveResources
+```powershell
+Remove-ClusterGroup -Name MyFileServer -Force -RemoveResources
 ```
 
-This example removes the clustered role named MyFileServer, without prompting for confirmation. All
-cluster resources in MyFileServer will be deleted.
+This example removes the clustered  role named `MyFileServer`, without prompting for confirmation.
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/failoverclusters/Remove-ClusterGroup.md:57]: Looks like an extra space snuck in.
```suggestion
This example removes the clustered role named `MyFileServer`, without prompting for confirmation.
```

File: docset/winserver2022-ps/failoverclusters/Remove-ClusterResourceDependency.md [hunk 3]
```diff
@@ -34,27 +34,26 @@ offline might vary.
...
-----                -----               -----               ------------ 
-cluster1FS          Online              cluster1FS          Network Name
+```powershell
+Remove-ClusterResourceDependency -Resource cluster1FS -Provider "IP Address 2001:4898:9:2:: (3)"
 ```
 
-This example removes the dependency between cluster resource cluster1FS and the resource named IP
-Address 2001:4898:9:2:: (3).
+This example removes the dependency between cluster resource `cluster1FS` and the resource named IP
+Address `2001:4898:9:2:: (3)`.
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/failoverclusters/Remove-ClusterResourceDependency.md:42]:
```suggestion
This example removes the dependency between cluster resource `cluster1FS` and the resource named
`IP Address 2001:4898:9:2:: (3)`.
```

File: docset/winserver2022-ps/failoverclusters/Remove-ClusterResourceDependency.md [hunk 4]
```diff
@@ -34,27 +34,26 @@ offline might vary.
 
 ### Example 1
 
-```
-PS C:\> Remove-ClusterResourceDependency -Resource cluster1FS -Provider "IP Address 2001:4898:9:2:: (3)"
-Name                State               Group               ResourceType 
-----                -----               -----               ------------ 
-cluster1FS          Online              cluster1FS          Network Name
+```powershell
+Remove-ClusterResourceDependency -Resource cluster1FS -Provider "IP Address 2001:4898:9:2:: (3)"
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/failoverclusters/Remove-ClusterResourceDependency.md:38]: Should this example use splatting like the one after it?
```suggestion
```powershell
$parameters = @{
    Resource = 'cluster1FS'
    Provider = 'IP Address 2001:4898:9:2:: (3)'
}
Remove-ClusterResourceDependency @parameters
```

File: docset/winserver2022-ps/failoverclusters/Remove-ClusterResourceDependency.md [hunk 5]
```diff
@@ -34,27 +34,26 @@ offline might vary.
...
+$parameters = @{
+    Provider = 'IP Address 2001:4898:9:2:: (3)'
+}
+Get-ClusterResource -Name cluster1FS | Remove-ClusterResourceDependency @parameters
 ```
 
-This example removes the dependency between the cluster resource named cluster1FS and the resource
-named IP Address 2001:4898:9:2:: (3).
+This example removes the dependency between the cluster resource named `cluster1FS` and the resource
+named IP Address `2001:4898:9:2:: (3)`. This example uses splatting to pass parameter values from
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/failoverclusters/Remove-ClusterResourceDependency.md:54]:
```suggestion
named `IP Address 2001:4898:9:2:: (3)`. This example uses splatting to pass parameter values from
```

File: docset/winserver2022-ps/failoverclusters/Set-ClusterOwnerNode.md [hunk 6]
```diff
@@ -32,21 +32,21 @@ failure of a resource or a clustered role.
...
 
-```
-PS C:\> Get-ClusterResource -Name "Cluster Disk 3" | Set-ClusterOwnerNode -Owners node1,node2
+```powershell
+Get-ClusterResource -Name "Cluster Disk 3" | Set-ClusterOwnerNode -Owners node1,node2
 ```
 
-This example sets the possible owners for cluster named Cluster Disk 3 on the local cluster to the
-nodes named node1 and node2.
+This example sets the possible owners for cluster named `Cluster Disk 3` on the local cluster to the
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/failoverclusters/Set-ClusterOwnerNode.md:39]: Here the docs say `possible` owners, and down on line 48 it says `preferred`.

File: docset/winserver2022-ps/failoverclusters/Set-ClusterParameter.md [hunk 7]
```diff
@@ -56,46 +56,21 @@ such as a resource, a group, or a network.
...
 
-```
-PS C:\> Get-ClusterResource -Name cluster1FS | Set-ClusterParameter -Name HostRecordTTL -Value 300
+```powershell
+Get-ClusterResource -Name "Cluster Disk 3" | Set-ClusterOwnerNode -Owners node1,node2
 ```
 
-This example configures the clustered resource called cluster1FS on the local cluster, by setting
-the value of HostRecordTTL to 300.
+This example sets the possible owners for cluster named `Cluster Disk 3` on the local cluster to the
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/failoverclusters/Set-ClusterParameter.md:63]: Same remark for possible vs preferred.

File: docset/winserver2022-ps/failoverclusters/Set-ClusterQuorum.md [hunk 8]
```diff
@@ -33,44 +33,45 @@ cluster configuration) or file share witness.
...
-using the disk resource at \\\\fileserver\fsw for the file share witness.
+using the disk resource at `\\fileserver\fsw` for the file share witness.
 
 ### Example 4
 
+```powershell
+$parameters = @{
+    CloudWitness = $true
+    AccountName = '<AzureStorageAccountName>'
+    AccessKey = '<AzureStorageAccountAccessKey>'
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/failoverclusters/Set-ClusterQuorum.md:66]:
```suggestion
$parameters = @{
    CloudWitness = $true
    AccountName  = '<AzureStorageAccountName>'
    AccessKey    = '<AzureStorageAccountAccessKey>'
}
```

File: docset/winserver2022-ps/failoverclusters/Set-ClusterResourceDependency.md [hunk 9]
```diff
@@ -35,28 +35,35 @@ deployments.
...
+This example makes the resource called cluster1FS12 dependent on `[IP Address 151.56.48.0]`.
 
 ### Example 2
 
-```
-C:\PS>Set-ClusterResourceDependency -Resource cluster1FS12 -Dependency "[IP Address 151.56.48.0] or [New IP Address]"
+```powershell
+$parameters = @{
+    Resource = 'cluster1FS12'
+    Dependency = '[IP Address 151.56.48.0]'
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/failoverclusters/Set-ClusterResourceDependency.md:49]: It looks like the `or` statement was accidentally removed.
```suggestion
    Dependency = '[IP Address 151.56.48.0] or [New IP Address]'
```

File: docset/winserver2022-ps/failoverclusters/Set-ClusterStorageSpacesDirectDisk.md [hunk 10]
```diff
@@ -34,14 +34,24 @@ S2D. Alternatively, you can run this cmdlet after you enable S2D.
...
 ### Example 1: Configure disks not to be claimed
 
-```
-PS C:\> Set-ClusterStorageSpacesDirectDisk -CimSession "K0619-C1.contoso.com" -CanBeClaimed:$False -PhysicalDiskIds "55CD2E404B75A3FC","50014EE05950DD7C"
+```powershell
+$parameters = @{
+    CimSession = 'K0619-C1.contoso.com'
+    CanBeClaimed = $False
+    PhysicalDiskIds = '55CD2E404B75A3FC',
+                      '50014EE05950DD7C'
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/failoverclusters/Set-ClusterStorageSpacesDirectDisk.md:42]: I'd recommend either using `@()` to group these entries or have them on a single line if that fits.

```suggestion
    PhysicalDiskIds = @(
        '55CD2E404B75A3FC'
        '50014EE05950DD7C'
    )
```

```suggestion
    PhysicalDiskIds = '55CD2E404B75A3FC', '50014EE05950DD7C'
```

**Review by michaeltlombardi — CHANGES_REQUESTED** [review]
Some minor changes and suggestions.

File: docset/winserver2022-ps/failoverclusters/Set-ClusterOwnerNode.md (hunk 6 above)
Comment (robinharwood) [reply docset/winserver2022-ps/failoverclusters/Set-ClusterOwnerNode.md:39]: This is correct. If the input object is a cluster resource, then the parameter applies the possible owner. Preferred owners only apply to cluster groups.

File: docset/winserver2022-ps/failoverclusters/Remove-ClusterCheckpoint.md (hunk 1 above)
Comment (robinharwood) [reply docset/winserver2022-ps/failoverclusters/Remove-ClusterCheckpoint.md:45]: Good point, plus the command didn't make sense. Updated.

**Comment by robinharwood** [comment]
Hi @michaeltlombardi - I think everything should be updated on this PR now. Please let me know if there is anything else.

Thanks,
Robin

**Review by michaeltlombardi — APPROVED** [review]

**Pull request merged**